src/legacy/core_plugins/vis_type_vislib/public/vislib/__tests__/response_handlers.js

```diff
@@ -21,6 +21,7 @@ import sinon from 'sinon';
 import ngMock from 'ng_mock';
 import expect from '@kbn/expect';
 
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
Leaving in unallowed import here because these tests won't be migrated to the new platform anyway